src/AppInstallerRepositoryCore/Rest/Schema/Json/InformationResponseDeserializer.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "Rest/Schema/IRestClient.h"
+#include <cpprest/json.h>
+#include "JsonHelper.h"
+#include "InformationResponseDeserializer.h"
+#include "CommonRestConstants.h"
+
+namespace AppInstaller::Repository::Rest::Schema::Json
+{
+    namespace
+    {
+        // Information response constants
+        constexpr std::string_view SourceIdentifier = "SourceIdentifier"sv;
+        constexpr std::string_view ServerSupportedVersions = "ServerSupportedVersions"sv;
+        constexpr std::string_view APIVersion = "APIVersion"sv;
+        constexpr std::string_view ObjectVersions = "ObjectVersions"sv;

```

Comment:
I don't understand what all of these versions are for.

Reply:
Updated based on offline conversation.

---
In reply to: [596390423](https://github.com/microsoft/winget-cli/pull/800#discussion_r596390423) [](ancestors = 596390423)